code-challenges.test.js

```diff
@@ -14,46 +14,140 @@
 
-// a) Create a test with expect statements for each of the variables provided.
-
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
-
-// b) Create the function that makes the test pass.
-
-// --------------------3) Create a function that takes in a string and logs the index of the first vowel.
+//Pseudocode:
+// input - build a jest test using the given variables. After building the test create a function.
+//Output- the function will cause the test to pass and return each letter in every word capitalized in the array.
+// create a test with the describe function being capitalizeWords. Within the describe method input take in an array of words and returns an array with all the words capitalized." For the expect method call upon function capitalizeWords with the values being the given variables randomNouns1 and randomNouns2. When logging just the test it comes back as undefined.
+// Create a function  using the variable capitalizeWords with array in the parameter. Make a return using array.map. When building the function method use word instead of string when using an array.(This was the mistake I was running into)  use word.charAt(0) to isolate the first letter in the string and use .toUpperCase() to capitalize it. next use slice to extract a section of the string and return it as a new string without modifying the original.
+//console.log the variable and call upon the parameter to display the new arrays in the terminal.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const vowelTester1 = "learn"
-// Expected output: 1
-const vowelTester2 = "academy"
-// Expected output: 0
-const vowelTester3 = "challenges"
-// Expected output: 2
+// const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+// // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+// const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
-// b) Create the function that makes the test pass.
+// describe(capitalizeWords, () => {
+//         it("take in an array of words and returns an array with all the words capitalized.", () => {
+//           expect(capitalizeWords(randomNouns1)).toString("Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"),
+//           expect(capitalizeWords(randomNouns2)).toString("Temperature", "Database", "Chopsticks", "Mango")
+//         }) 
+//          })   //Output: ReferenceError: capitalizeWords is not defined //Test Suites: 1 failed, 1 total
+         //Tests:       0 total
+       
+         
+// // // b) Create the function that makes the test pass.
+
+// function capitalizeWords (array) {
```

Comment:
Try to keep your declarations to const for that global scope to carry your functions throughout your file.